packages/app/src/game/module/board/overlays/readyOverlay.tsx

```diff
@@ -1,41 +1,16 @@
 import { usePlayerId } from "packages/app/src/auth";
-import { GamePhase, PlayerListPlayer, StreakType } from "packages/models/lib";
+import { GamePhase, PlayerListPlayer } from "packages/models/lib";
 import * as React from "react";
 import { useSelector } from "react-redux";
+import { ProgressBar } from "../../../../display";
 import { AppState } from "../../../../store";
+import { PlayerPicture } from "../../playerList/components/playerPicture";
```

Comment:
Something worth thinking about here - importing from a very very deeply nested "cousin" like this (you went up a few levels and then deeply into another branch) is often a sign that some concerns are getting a bit crossed, and it might be time to lift some things up into a higher level.

This specific instance is probably a good sign that we need to think about creating a "component library" which contains basic components for common things, e.g. player picture/title, progress/health bars, buttons, etc